src/microbe_stage/editor/CellEditorComponent.cs

```diff
@@ -1542,7 +1542,7 @@ private void StartAutoEvoPrediction()
     {
         biome ??= Editor.CurrentPatch.Biome;
 
-        var energyBalance = ProcessSystem.ComputeEnergyBalance(organelles, biome, membrane, true,
+        var energyBalance = ProcessSystem.ComputeEnergyBalance(organelles, biome, membrane, true, true,
```

Comment:
Isn't this being always true wrong?

Reply:
If I flip this to false, and remove the bool check from the bool in Microbe.Interior, it reverses the functionality and gives bacteria the osmoregulation bonus, and not eukaryotes.

Reply:
I didn't mean flipping this. Just that this looks like this is the editor prediction, which means that this should use the current editor state to determine if the player is currently bacteria or not.

Reply:
I admit I don't entirely understand how the editor prediction works. I just used the Isplayer boolean as an example until it worked as I intended.

It seems to work perfectly fine, I just assumed that the true is default(?) but I do not know for sure. I'm also not entirely sure how to change it to your liking without breaking it. Sorry....

Reply:
Ok I *think* I have done as you asked.

Reply:
It's more towards the right direction now, but there's still some very unexpected parts to the change.